include/zenoh-pico/api/liveliness.h

```diff
@@ -0,0 +1,153 @@
+//
+// Copyright (c) 2024 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+#ifndef INCLUDE_ZENOH_PICO_API_LIVELINESS_H
+#define INCLUDE_ZENOH_PICO_API_LIVELINESS_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "olv_macros.h"
+#include "zenoh-pico/api/types.h"
+#include "zenoh-pico/protocol/core.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
+    uint32_t _id;
+    _z_keyexpr_t _key;
+    _z_session_weak_t _zn;
+} _z_liveliness_token_t;
+
+_Z_OWNED_TYPE_VALUE(_z_liveliness_token_t, liveliness_token)
+_Z_OWNED_FUNCTIONS_DEF(liveliness_token)
+
+#if Z_FEATURE_LIVELINESS == 1
+
+/**************** Liveliness Token ****************/
+
+/**
+ * The options for :c:func:`z_liveliness_declare_token()`.
+ */
+typedef struct z_liveliness_token_options_t_t {
```

Comment:
I guess there is a query/replace error here:
Should be `z_liveliness_token_options_t`and not `z_liveliness_token_options_t_t`.

Reply:
It was a typo, thanks!